homeassistant/helpers/update_coordinator.py

```diff
@@ -239,7 +249,7 @@ async def _async_refresh(  # noqa: C901
 
         self._debounced_refresh.async_cancel()
 
-        if scheduled and self.hass.is_stopping:
+        if self._shutdown_requested or scheduled and self.hass.is_stopping:
```

Comment:
Note;
For the Debouncer, I went with a `RuntimeError` if there are subsequent calls.
Here, I went with just ignoring the call.